README.md

```diff
@@ -72,6 +72,19 @@ You can even install desired versions of languages and the global version:
       - 2.6.4
 ```
 
+It's also possible to configure the location for asdf in case asdf itself was
+installed as part of the dotbot install process. This will cause the plugin to
+source the provided script before every asdf command.
+
+```yaml
+- asdf:
+  - asdf_path: /opt/asdf-vm/asdf.sh
```

Comment:
I think we need to mention how order is important here.

Reply:
Fair, I'll add it in

Reply:
It's not important, I extract the config before I loop through the list. The only important thing is that multiple asdf_path-nodes beyond the first will have no effect. I did it like this so I wouldn't break people's configs by changing the base data structure.